sqlc-database/Makefile

```diff
@@ -0,0 +1,2 @@
+sqlc:
```

Comment:
We don't use makefiles in the other examples, would be better to just mention in the README for this example to use `sqlc generate`